apps.awesim.org/apps/myjobs/templates/Basic_Python_Serial_Job_Owens/pbs_script.sh

```diff
@@ -1,10 +1,10 @@
 #PBS -N ondemand/sys/myjobs/basic_python_serial
 #PBS -j oe
 #PBS -l walltime=0:10:00
-#PBS -l nodes=1:ppn=28
+#PBS -l nodes=1:ppn=1
```

Comment:
Given the trivial nature of this script changing `ppn` from 28 to 1 is a good decision.